src/BCFKeyValueData.cc

```diff
@@ -195,6 +199,15 @@ Status BCFKeyValueData::InitializeDB(KeyValue::DB* db,
                                      int interval_len) {
     Status s;
 
+    // some basic sanity checks
+    if (contigs.size() > MAX_NUM_CONTIGS_PER_GVCF)
+        return Status::Invalid("Too many contigs (", std::to_string(contigs.size()) + ")");
```

Comment:
The two strings aren't used quite as this seems to expect. https://github.com/dnanexus-rnd/GLnexus/blob/orodeh_gvcf_ingest_validation/include/types.h#L82